sdk/openai/azure-ai-openai/CHANGELOG.md

```diff
@@ -1,212 +1,5 @@
 # Release History
 
-## 1.0.0-beta.9 (Unreleased)
```

Comment:
This is openai changelog and should not be modified.

Reply:
Rollback the change.